rules/defaults.yaml

```diff
@@ -924,8 +895,6 @@
 
 ^SoldierFearlessVeryHeavy:
 	Inherits: ^SoldierFearlessHeavy
```

Comment:
I guess that we can remove ^SoliderFearlessVeryHeavy as it's exact copy of ^SoliderFearlessHeavy